packages/synapse-interface/pages/stake/StakeCard.tsx

```diff
@@ -59,8 +60,8 @@ const StakeCard = ({ address, chainId, pool }: StakeCardProps) => {
   })
 
   useEffect(() => {
-    if (!address || !chainId || !stakingPoolId) return
-    getStakedBalance(`0x${address.slice(2)}`, chainId, stakingPoolId)
+    if (!address || !chainId || stakingPoolId === null) return
```

Comment:
nice fix

@nautsimon  @bigboydiamonds  can we please be more explicit w/ these type checks going forward? If zero chain ids existed, we could easily make the same mistake there.